src/components/DataViewer/DataViewer.tsx

```diff
@@ -15,45 +17,70 @@ const StyledModal = styled(Modal)`
     left: 50%;
     transform: translate(-50%, -50%);
     border: 0;
-    border-radius: 5px;
+    border-radius: 2em;
     outline: none;
     padding: 0;
     width: 80%;
     max-height: 80%;
-    background: rgb(255, 255, 255);
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
     box-shadow: rgb(200, 200, 200) 0 2px 10px;
     overflow: hidden;
+    max-width: 1000px;
 `;
-const Header = styled.h3`
+
+const Header = styled.div`
     display: flex;
     justify-content: space-between;
     align-items: center;
-    background-color: #eeeeee;
-    padding: 0.5rem;
-    margin-top: 0;
+    color: ${(props) => props.theme.secondaryForegroundColor};
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}A0;
+    font-size: 1.5em;
+    font-weight: bolder;
+    padding: 0.75em 1em 0.75em 1em;
 `;
+
 const Content = styled.div`
-    overflow: scroll;
-    margin: 1.5rem;
-    margin-top: 1rem;
+    overflow: auto;
+    margin: 1rem 1.5rem 1.5rem 1.5rem;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
 `;
+
 const Key = styled.div`
-    color: grey;
-    font-size: small;
+    color: ${(props) => props.theme.foregroundColor};
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}40;
+    display: inline-block;
+    font-size: medium;
+    border-radius: 0.7em;
+    padding: 0.2em 0.5em 0.2em 0.5em;
 `;
+
 const Value = styled.div`
     font-size: medium;
+    padding: 0.2em 0.5em 0.2em 0.5em;
```

Comment:
Same here

(I think using the shorthand is good for consistency, but it may not be necessary)

Reply:
agree on shorthand being a good thing!
these might be remnants from testing different values, but the final production code should definitely have shorter CSS where possible :)

Reply:
Refactored!